extension/js/common/core/att.ts

```diff
@@ -97,7 +99,7 @@ export class Att {
     }
   }
 
-  public static pgpNamePatterns = () => ['*.pgp', '*.gpg', '*.asc', 'noname', 'message', 'PGPMIME version identification', ''];
+  public static readonly attachmentsPattern = /^(((cryptup|flowcrypt)-backup-[a-z]+\.key)|(.+\.pgp)|(.+\.gpg)|(.+\.asc)|(noname)|(message)|(PGPMIME version identification)|())$/gm;
```

Comment:
Looks good, it's a complicated pattern but it simplifies the logic overall, thanks!